<?xml version="1.0"?>
<launch>
    <remap from="/depth/points" to="/sensor_point_cloud"/>
    <param name="camera_frame_id" type="string" value="tool0" />
    <node name="ensenso_publisher_node" pkg="ensenso_publisher" type="ensenso_publisher_node" output="screen">
        <param name="FlexView" type="bool" value="false" />
        <param name="FlexViewImages" type="int" value="4" />
    </node>
</launch>
